docker/Dockerfile_ubuntu_18.04

```diff
@@ -0,0 +1,27 @@
+FROM nvidia/cudagl:9.2-runtime-ubuntu18.04
+
+RUN apt-get update && apt-get install -y --no-install-recommends \
+    python3 python3-dev ipython3 module-init-tools curl build-essential python3-pip
+
+RUN pip3 install -U pip setuptools wheel
+
+RUN pip3 install numpy posix_ipc holodeck pytest opencv-python
+
+RUN adduser --disabled-password --gecos "" holodeckuser
+
+RUN echo holodeckuser:holodeck | chpasswd
+
+# This should be COPY ../ but docker doesn't allow copying files outside the context
+# for this to copy the project files either run the build command in this directory with the
+# previous directory as the context: docker build -t pccl/holodeck[:tag] -f ./Dockerfile ..
+# or run it from the parent directory and provide the docekr file location
+# docker build -t pccl/holodeck[:tag] -f ./docker/Dockerfile .
+COPY ./ /home/holodeckuser/source/holodeck/
+
+USER holodeckuser
+
+RUN python3 -c 'import holodeck; holodeck.install("DefaultWorlds")'
```

Comment:
Instead of having different docker images for different versions of ubuntu I vote we have different docker images for different packages. (Let's just use 18.04?)

Maybe:

- base (no packages)
- all (all packages)
- `DefaultWorlds`
- `Dexterity`

You also don't need to have multiple, duplicated dockerfiles for each. I would have dockerfile that installs holodeck and no worlds ("base" image), then other dockerfiles that would call `FROM` with that base image. 

The child dockerfiles would basically only install different packages. You would need to amend your build script to build and tag the parent base image first.

Reply:
Done. Except I decided not to have `all` for now.